content/docs/command-reference/gc.md

```diff
@@ -120,9 +121,31 @@ project we want to clear.
   them (each project is a path) to keep data that is currently referenced from
   them.
 
+- `--not-in-remote` - keep cached objects that are _not_ in the remote. This
+  will remove the objects from the local cache that have been pushed and are
+  present in the remote.
+
+  For objects using the
+  [`remote` field](/doc/user-guide/project-structure/dvc-files#output-entries),
+  the check will be against that remote.
+
+  For objects not using the
+  [`remote` field](/doc/user-guide/project-structure/dvc-files#output-entries),
+  the check will be against the default remote unless a specific one is given
+  with `-r`.
+
 - `-c`, `--cloud` - remove files in remote storage in addition to local cache.
-  **This option is dangerous.** The default remote is used unless a specific one
-  is given with `-r`.
+
+  **This option is dangerous.**
+
+  For objects using the
+  [`remote` field](/doc/user-guide/project-structure/dvc-files#output-entries),
+  the check will be against that remote.
```

Comment:
WDYT about adding something like this?

```
Any files not using the `remote` field may be deleted from that remote.
```

And then below something like this?

```
Any files using the `remote` field may be deleted from the default remote.
```